src/test/java/seedu/address/logic/commands/AddProcCommandTest.java

```diff
@@ -59,6 +68,33 @@ public void execute_properIndexWithProperProcedure_success() {
         assertCommandSuccess(addProcCommand, newModel, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_autosortProcedureWhenAdded_success() {
+        Random rand = new Random();
+        RandomProcedureBuilder randomProcedureBuilder = new RandomProcedureBuilder();
+        List<Procedure> expectedProcList = new ArrayList<>();
+        List<Procedure> currentProcList = new ArrayList<>();
+        AddProcCommand addProcCommand = new AddProcCommand(INDEX_FIRST_CLIENT, toBeAddedProcedure);
+
+        // Adding in a sorted manner
+        expectedProcList.add(toBeAddedProcedure);
+        for (int i = 0; i < 3; i++) {
```

Comment:
Good coding standard thus far, but would it be good to create a static variable in order to reduce magic words in the for loop?

Reply:
Good idea. Resolved in newer update.